requirements.txt

```diff
@@ -1,52 +1,52 @@
 # Upstream is missing Python 3 patches
 git+https://github.com/JelteF/python-memcached@patch-1#egg=python-memcached==1.58
 markdown2==2.3.8                # textarea text formatting
 pytz                            # Timezone lib
-stripe==2.29.4                  # Stripe payment
+stripe==2.35.1                  # Stripe payment
 icalendar==4.0.3                # iCalendar generation
-google-api-python-client==1.7.9
+google-api-python-client==1.7.11
 celery==4.3.0                   # Async task runner
 
 # third party deps
 django-filebrowser==3.11.1      # File uploading
 git+https://github.com/dotKom/django-chunks.git@e6bf109d4fd286964024c140ac1b8f3d86713540#egg=django-chunks==0.4
 django-crispy-forms==1.7.2      # nice forms
-django-extensions==2.1.7        # nice extra commands for debugging, etc
+django-extensions==2.2.1       # nice extra commands for debugging, etc
 django-dynamic-fixture==2.0.0   # Dynamic fixtures for models
-django-recaptcha==2.0.4         # Google ReCaptcha
+django-recaptcha==2.0.5         # Google ReCaptcha
 django-oauth-toolkit==1.2.0    # OAuth2 authentication support
 django-watson==1.5.2            # Indexed model search lib
 django-reversion==3.0.4        # Model version history with middleware hooks to all post_save signals
-django-guardian==1.5.1          # Per Object permissions framework
+django-guardian==2.0.0          # Per Object permissions framework
 django-taggit==1.1.0           # Generic multi-model tagging library
 django-taggit-serializer==0.1.7 # REST Framework serializers for Django-taggit
-APScheduler==3.6.0              # Scheduler
+APScheduler==3.6.1              # Scheduler
 git+https://github.com/dotkom/redwine.git@c26e12003a26bd4ef15c10d55358c24f96b3d132#egg=redwine==1.3.0
 reportlab==3.5.23
 pdfdocument==3.3
-Unidecode==1.0.23               # Translates every unicode symbol to the closest ascii. online_mail_usernames
+Unidecode==1.1.1               # Translates every unicode symbol to the closest ascii. online_mail_usernames
 django-markdown-deux==1.0.5
-django-cors-headers==3.0.2      # Enable CORS for Nibble
+django-cors-headers==3.1.0      # Enable CORS for Nibble
 git+https://github.com/oleast/django-datetime-widget.git@95a5ba4738853d1ed4081e1e33e1600cba984aad#egg=django-datetime-widget==0.9.4
 django-webpack-loader==0.6.0    # Integration with webpack
 dj-database-url==0.5.0          # Allows to configure databases using URLs
 python-decouple==3.1            # Configuration and settings management
 django-oidc-provider==0.7.0     # OpenID Connect Provider
-docutils==0.14                  # Allows django-admindocs to generate documentation
+docutils==0.15.2                  # Allows django-admindocs to generate documentation
 requests-oauthlib==1.2.0        # Simplify OAuth2 connections to external apps
-oic==0.15.1			            # OpenID Connect Client
+oic==1.0.1			            # OpenID Connect Client
```

Comment:
This upgrades a major version of the library, does this affect us?

Reply:
According to their GitHub they only dropped alle support for python 3.5 or lower and since we are on 3.7 we should be fine. The rest should be a bunch of fixes

Reply:
Ok, goodie. Let's run it to dev ❤️